src/components/ComponentGallery/index.css

```diff
@@ -0,0 +1,50 @@
+@import "@ukic/fonts/dist/fonts.css";
```

Comment:
How come we're importing these again? I imagine we have these somewhere else. Likely this could be causing the bullet issue